Multisig/UI/UI Library/Overlays/BottomOverlayView.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  BottomOverlayView.swift
+//  Multisig
+//
+//  Created by Andrey Scherbovich on 28.05.20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import SwiftUI
+
+struct BottomOverlayView<Content>: View where Content: View {
+    private var isPresented: Binding<Bool>
+    private var content: Content
+
+    private let cardBackgroundColor = Color.white
+
+    public init(isPresented: Binding<Bool>,
+               @ViewBuilder content: () -> Content) {
+        self.content = content()
+        self.isPresented = isPresented
+    }
+
+    var body: some View {
+        ZStack(alignment: .bottom) {
+            if isPresented.wrappedValue {
+                SemitransparentBackgroundView()
+                    .transition(AnyTransition.opacity.animation(.easeInOut))
+                    .onTapGesture {
+                        withAnimation {
+                            self.isPresented.wrappedValue.toggle()
+                        }
+                    }
```

Comment:
this seems to be very similar to the "PopUpView", can this bottom overaly view used as is to implement the popup view?

Reply:
Do you mean inheriting PopUpView from BottomOverlayView or from some generic OverlayView?

Reply:
I mean that the popup view is almost the same as bottom overlay view (and other way around), so maybe it can be implemented using the bottom overlay view

Reply:
I would leave them as they are for a while.